test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/ModelMetadataProviderTest.cs

```diff
@@ -663,26 +663,11 @@ public void IsRequired_DataMemberIsRequiredFalse_FalseForReferenceType()
                 nameof(ClassWithDataMemberIsRequiredFalse.StringProperty));
 
             // Assert
-            Assert.False(metadata.IsRequired);
-        }
-
-        [Fact]
-        public void IsRequired_DataMemberIsRequiredFalse_TrueForValueType()
-        {
-            // Arrange
-            var metadataProvider = TestModelMetadataProvider.CreateDefaultProvider();
-
-            // Act
-            var metadata = metadataProvider.GetMetadataForProperty(
-                typeof(ClassWithDataMemberIsRequiredFalse),
-                nameof(ClassWithDataMemberIsRequiredFalse.IntProperty));
-
-            // Assert
-            Assert.True(metadata.IsRequired);
+            Assert.False(metadata.IsBindingRequired);
```

Comment:
`IsBindingRequired` defaults to false (not computed based on type like `IsRequired`) - so these tests are obsolete.